geode-core/src/distributedTest/java/org/apache/geode/cache/query/dunit/ResourceManagerWithQueryMonitorDUnitTest.java

```diff
@@ -997,274 +850,185 @@ private void handleException(Exception e, boolean hitCriticalThreshold,
-          cache.MAX_QUERY_EXECUTION_TIME = queryTimeout;
-        } else {
-          cache.MAX_QUERY_EXECUTION_TIME = -1;
-        }
+      cache.MAX_QUERY_EXECUTION_TIME = queryTimeout;
 
-        if (criticalThreshold != 0) {
-          InternalResourceManager resourceManager =
-              (InternalResourceManager) cache.getResourceManager();
-          HeapMemoryMonitor heapMonitor = resourceManager.getHeapMonitor();
-          heapMonitor.setTestMaxMemoryBytes(1000);
-          HeapMemoryMonitor.setTestBytesUsedForThresholdSet(NORMAL_HEAP_USED);
-          resourceManager.setCriticalHeapPercentage(criticalThreshold);
-        }
+      InternalResourceManager resourceManager =
+          (InternalResourceManager) cache.getResourceManager();
+      HeapMemoryMonitor heapMonitor = resourceManager.getHeapMonitor();
+      heapMonitor.setTestMaxMemoryBytes(1000);
+      HeapMemoryMonitor.setTestBytesUsedForThresholdSet(NORMAL_HEAP_USED);
+      resourceManager.setCriticalHeapPercentage(85);
 
-        AttributesFactory factory = new AttributesFactory();
-        if (createPR) {
-          PartitionAttributesFactory paf = new PartitionAttributesFactory();
-          paf.setRedundantCopies(prRedundancy);
-          paf.setTotalNumBuckets(11);
-          factory.setPartitionAttributes(paf.create());
-        } else {
-          factory.setScope(Scope.DISTRIBUTED_ACK);
-          factory.setDataPolicy(DataPolicy.REPLICATE);
-        }
-        Region region = createRootRegion(regionName, factory.create());
-        if (createPR) {
-          assertTrue(region instanceof PartitionedRegion);
-        } else {
-          assertTrue(region instanceof DistributedRegion);
-        }
-        CacheServer cacheServer = getCache().addCacheServer();
-        cacheServer.setPort(port);
-        cacheServer.start();
-
-        return null;
+      RegionFactory<String, Portfolio> factory = cache.createRegionFactory();
+      if (createPR) {
+        PartitionAttributesFactory paf = new PartitionAttributesFactory();
+        paf.setRedundantCopies(0);
+        paf.setTotalNumBuckets(11);
+        factory.setPartitionAttributes(paf.create());
+      } else {
+        factory.setScope(Scope.DISTRIBUTED_ACK);
+        factory.setDataPolicy(DataPolicy.REPLICATE);
       }
-    });
-  }
-
-  private void startClient(VM client, final VM server, final int port, final String regionName) {
-
-    client.invoke(new CacheSerializableRunnable("Start client") {
-      @Override
-      public void run2() throws CacheException {
-        Properties props = getClientProps();
-        getSystem(props);
-
-        final ClientCacheFactory ccf = new ClientCacheFactory(props);
-        ccf.addPoolServer(NetworkUtils.getServerHostName(server.getHost()), port);
-        ClientCache cache = (ClientCache) getClientCache(ccf);
+      Region region = createRootRegion("portfolios", factory);
+      if (createPR) {
+        assertThat(region instanceof PartitionedRegion).isTrue();
```

Comment:
Or:
```
assertThat(region).isInstanceOf(PartitionedRegion.class);
```
Although I would just delete this whole if-else block because the assertions are worthless.

Reply:
Done.